ppx/dune

```diff
@@ -4,7 +4,7 @@
  (synopsis "[@@deriving rpc]")
  (kind ppx_deriver)
  (ppx_runtime_libraries rpclib)
- (libraries ppxlib)
+ (libraries base ppxlib)
```

Comment:
Can you remove base here? It does not seem that any of your changes are requiring its reintroduction

Reply:
My bad, I added this because `ppx_deriving_rpc` used to rely on base transitively through ppxlib and we dropped the base dependency a while ago.

I'll remove it!